db/migrations/0002_alter_player_email.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.8 on 2023-12-11 10:35
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='email',
+            field=models.EmailField(max_length=255),
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app in a pull request in Django projects to not overload the `migrations` folder. Include one migration with all necessary changes to your pull request.